src/MusicTheory/PitchClass.elm

```diff
@@ -97,8 +78,8 @@ pitchClass l acc =
 
 -}
 fromTuple : ( Letter, Accidental ) -> PitchClass
```

Comment:
Not exactly necessary. Could be convenient for clients, though. But we could remove it until we proof that we need it.

Reply:
See later comments as well.

Note that `fromTuple` is useful here: https://github.com/elm-music/music-theory/blob/feature/start/pitch-class-enharmonics/tests/PitchClassTests.elm#L119

This test should be moved to `SpellingTests`. Anyway it shows how it can be useful to be able to easily go back to a `PitchClass` representation from a `PitchClassSpelling` representation.